lib/wit.js

```diff
@@ -57,76 +62,89 @@ function Wit(opts) {
     ;
   };
 
-  const continueRunActions = (sessionId, message, prevContext, i) => {
-    return (json) => {
+  const continueRunActions = (sessionId, index, message, prevContext, i) => {
+    return ({action, entities, msg, quickreplies, type}) => {
```

Comment:
I'd rather keep it as it was before to maximize similarity with Ruby and Python SDKs (very similar today)